.travis.yml

```diff
@@ -8,14 +8,14 @@ os:
 language: go
 
 go:
-  - 1.5.2
+  - 1.8
 
 env:
   - TEST_VERBOSE=1
 
 script:
-  - cd sharness
-  - make
+  - make -C sharness
+  - go test ./ipfs-5-to-6/migration/
```

Comment:
I can add a test target, but "go test ./..." fails locally for me, which is why I only added my tests.